src/pages/settings/SettingForm.tsx

```diff
@@ -133,9 +164,19 @@ const SettingForm: FC<Props> = ({ configField, value, isLast }) => {
                   : "You do not have permission to edit server configuration"
               }
             >
-              <div className="readmode-value u-truncate">
-                {getReadModeValue()}
-              </div>
+              {isClustered && isMaasMachine ? (
```

Comment:
```suggestion
              {isClusteredInput ? (
```
We should simplify and generalize here.